backend/src/config/sqlite.js

```
@@ -0,0 +1,102 @@
+  async transaction(callback) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        await this.query('BEGIN TRANSACTION');
+        const result = await callback(this);
+        await this.query('COMMIT');
+        resolve(result);
+      } catch (error) {
+        await this.query('ROLLBACK');
+        reject(error);
+      }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)